nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/content-viewer/feature/content-viewer.component.html

```diff
@@ -0,0 +1,49 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div class="content-viewer-background h-screen flex flex-col gap-y-4 p-4">
+    <div class="flex justify-between items-center">
+        <div>
+            <img ngSrc="assets/icons/nifi-logo-about.svg" priority width="150" height="64" alt="NiFi Logo" />
+        </div>
+        <form [formGroup]="viewerForm">
+            <mat-form-field subscriptSizing="dynamic" class="w-72">
+                <mat-label>View</mat-label>
+                <mat-select formControlName="viewAs" (selectionChange)="viewAsChanged($event)">
```

Comment:
Consider conditionally setting the panelWidth of this select between `auto` and `null` depending on if there are any types of viewers with multiple options.
> If set to auto, the panel will match the trigger width. If set to null or an empty string, the panel will grow to match the longest option's text.

If the case where there are multiple content viewers for the same display name, the displayed value is really long and wraps. It makes it challenging to read.

<img width="321" alt="Screenshot 2024-08-30 at 1 34 41 PM" src="https://github.com/user-attachments/assets/c2378e36-b586-471f-885e-7eb2d976a0b6">